dags/glam_dev.py

```diff
@@ -0,0 +1,444 @@
+"""
+DEV Desktop ETL for https://glam.telemetry.mozilla.org/
+
+Generates and runs a series of BQ queries, see
+[bigquery_etl/glam](https://github.com/mozilla/bigquery-etl/tree/main/bigquery_etl/glam)
+in bigquery-etl and the
+[glam_subdags](https://github.com/mozilla/telemetry-airflow/tree/main/dags/glam_subdags)
+in telemetry-airflow.
+"""
+
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from operators.gcp_container_operator import GKENatPodOperator
+from operators.task_sensor import ExternalTaskCompletedSensor
+from airflow.models import Variable
+from airflow.operators.subdag_operator import SubDagOperator
+
+from glam_subdags.extract import extracts_subdag, extract_user_counts
+from glam_subdags.histograms import histogram_aggregates_subdag
+from glam_subdags.general import repeated_subdag
+from glam_subdags.generate_query import generate_and_run_desktop_query
+from utils.gcp import bigquery_etl_query, gke_command
+from utils.tags import Tag
+
+
+prod_project_id = "moz-fx-data-shared-prod"
+prod_dataset_id = "telemetry_derived"
+dev_dataset_id = "dev_telemetry_derived"
+tmp_project = "moz-fx-data-shared-prod"  # for temporary tables in analysis dataset
+default_args = {
+    "owner": "efilho@mozilla.com",
+    "depends_on_past": False,
+    "start_date": datetime(2019, 10, 22),
+    "email": [
+        "telemetry-alerts@mozilla.com",
```

Comment:
Do we need to send a notification to telemetry-alerts?

Reply:
removed